include/rtc/rtc.h

```diff
@@ -41,6 +41,9 @@ typedef enum {
 	RTC_GATHERING_COMPLETE = 2
 } rtc_gathering_state_t;
 
+typedef enum { none, fatal, error, warning, info, debug, verbose } LogLevel;
+
+void rtcInitLogger(LogLevel logLevel, plog::IAppender *appender = NULL);
```

Comment:
This part is dedicated to the C-compatible API, therefore you'd need another C++ function in the `rtc` namespace called by this one. I'll add it on top of your changes and merge everything.

Reply:
I don't see any problem about that. 
Your choice.